### ISSUE TITLE
Version: 1.0.7 Highlighter and Auto completion

### ISSUE THREAD

**Issue by nacr** [henjue/vue-for-idea#37]
Hello,

After updating to version 1.0.7 Highlighter and Auto completion are not working. 

Using;
PhpStorm 2016.1.2
Build #PS-145.1616, built on May 24, 2016
JRE: 1.8.0_77-b03 x86
JVM: Java HotSpot(TM) Server VM by Oracle Corporation

**Comment by springeye** [comment]
have more error logs?

**Comment by agileago** [comment]
+1

**Comment by deckvig** [comment]
+1 PhpStorm 2016.1.2
![2016-06-14 22 11 46](https://cloud.githubusercontent.com/assets/15033627/16045887/04086650-327d-11e6-8361-b441e2d4a184.png)

**Comment by nacr** [comment]
sorry no error log.

**Comment by springeye** [comment]
I guess when there is no install coffeescript plugins

**Comment by nacr** [comment]
can you rephrase your last comment?

**Comment by agileago** [comment]
i have enabled coffee plugin. but it is still not work

**Comment by PascalAnimateur** [comment]
Same problem here, the script portion of .vue files does not get syntax highlighting and code completion.

**Comment by springeye** [comment]
@PascalAnimateur add lang attr

**Comment by PascalAnimateur** [comment]
Can confirm using `<script lang="javascript">` solves the problem, but could this be detected as the default language?

**Comment by likeadeckofcards** [comment]
What changed that caused the highlighting to not be active anymore?

**Comment by PascaleBeier** [comment]
.editorconfig doesn't apply as well when defining script lang="javascript". Pretty meh :(

I'm getting the following error using vue-cli webpack as well:

```
Error: Cannot resolve module 'javascript'
```

**Comment by Ryqsky** [comment]
+1

**Comment by springeye** [comment]
@PascalAnimateur This version uses the default text, and the next version I will add the default Lang (template, script, style).

**Comment by springeye** [comment]
@PascaleBeier what???

**Comment by springeye** [comment]
![2](https://cloud.githubusercontent.com/assets/1314740/16068317/a6b0596a-32f7-11e6-8148-2b8ef90d8794.png) 
![3](https://cloud.githubusercontent.com/assets/1314740/16068388/04257a26-32f8-11e6-8c70-06ecb9c1279f.png)
@nacr-dev @agileago @deckvig  I don't have any problems here.

**Comment by springeye** [comment]
I'm sorry. Forgot to tell you that the 1.0.7 version must specify the lang attribute for template,script,style tag.(Currently template only supports HTML highlight)
@nacr-dev @agileago @deckvig @PascalAnimateur @infernobass7 @Ryqsky

**Comment by enso-san** [comment]
Adding the lang attribute fixed the highlighter, but brokes laravel elixir and browserify.
With the lang-attribute its no longer possible to compile scripts and components:
`Browserify Failed!: 'import' and 'export' may appear only with 'sourceType: module' (19:0)`

**Comment by springeye** [comment]
@enso-media Could you provide a screenshot, thank you。Join the chat [![Join the chat at https://gitter.im/henjue/vue-for-idea](https://badges.gitter.im/henjue/vue-for-idea.svg)](https://gitter.im/henjue/vue-for-idea?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

**Comment by enso-san** [comment]
![screen](https://cloud.githubusercontent.com/assets/19945920/16071331/0caeb7ce-32db-11e6-81f4-b6adeedec7f3.png)

**Comment by Ryqsky** [comment]
How long will it take to release next versions ? I am supporter of this project and I rely on this project when I'm writing code.
Thank you.

**Comment by PascaleBeier** [comment]
@henjue I am having the same issue @enso-media is facing with browserify - only that I'm using webpack.

**Comment by springeye** [comment]
@enso-media @Ryqsky  I don't think the problem is vue-for-idea. Suggest you look  [https://github.com/babel/babelify/issues/136](https://github.com/babel/babelify/issues/136)

**Comment by jsl9208** [comment]
Vue-loader can't resolve style tag with lang="scss", and there are too many annoying error highlighting when I write scss in lang="sass" tag now. What can I do to solve this problem?

**Comment by enso-san** [comment]
Browserify still works if those attributes are removed in the components.
Its not documented or best practice to use a lang-attribute in vue component tags.
So maybe its not a specific vue-for-idea bug, however the lang-attribute is not supposed to be there.
Downgrading for now, as everything worked as expected before...

**Comment by springeye** [comment]
@enso-media see https://github.com/vuejs/vue-loader ;vue-loader is support lang attribute

**Comment by bhulsman** [comment]
I downgraded to 1.0.6; https://plugins.jetbrains.com/plugin/8123?pr=&showAllUpdates=true

Currently I don't have time to figure this out and while the new highlighting is a lot better, none (without the lang attribute) is worse.

**Comment by springeye** [comment]
@jsl9208 You tell me your error message?

**Comment by jsl9208** [comment]
@henjue sry, it's not a plugin bug, I find a solution for webpack+vue-loader to compile lang=scss tag: 

```
vue: {
    loaders: {
      scss: 'style!css!sass'
    }
}
```

**Comment by springeye** [comment]
update version to 1.0.71.default use css and javascript。

**Comment by springeye** [comment]
support formatter and commenter on version 1.0.71

**Comment by murbanowicz** [comment]
I think it should be re-opened.
The issue is still preset or again, I do not know. I have 1.0.73 on one computer and alpha on another. The same issue - autocompletion does not work, HTML tags are not automatically closed etc. IT's making coding components really hard.

**Comment by springeye** [comment]
@mariaczi You add the lang attribute?My test is normal.

**Comment by murbanowicz** [comment]
What you mean?

wt., 21 cze 2016, 05:40 użytkownik henjue notifications@github.com
napisał:

**Comment by murbanowicz** [comment]
Autocompletion and closing tags are not working in all - HTML template, SCRIPT and STYLE.
